client/starwhale/base/cloud.py

```diff
@@ -296,11 +296,9 @@ def _fetch_bundle_all_list(
         uri_typ: ResourceType,
         page: int = DEFAULT_PAGE_IDX,
         size: int = DEFAULT_PAGE_SIZE,
-        filter_dict: t.Optional[t.Dict[str, t.Any]] = None,
+        filter_dict: t.Optional[t.List[str]] = None,
```

Comment:
`filter_dict`  is not used in the following function scope,  does this meet expections?

Reply:
All of the API impl do not support filter for now, I will support the filter in the next pr (including the local list filter)